### PR TITLE
docs(stats): improve documentation on update groups env names (config migration guide)

### PR DESCRIPTION
**Pull request by InoMurko**

### CODE REVIEW

File: stats/config/migration/m20240618_080000_update_groups_and_separate_layout/README.md [hunk 1]
```diff
@@ -73,8 +73,17 @@ Counters:
 - Old name - `STATS_CHARTS__COUNTERS__<NAME>__UPDATE_SCHEDULE`
 - New name - `STATS_UPDATE_GROUPS__SCHEDULES__<GROUP_NAME>`
 - Additional info - Schedule will be overridden for the whole group. `<GROUP_NAME>` is the name of update group
-that contains the `<NAME>` chart. It can be found in [update_groups.rs](../../../stats-server/src/update_groups.rs) or
-in [split.py (variable `update_groups_mapping`)](./split.py)
+that contains the `<NAME>` chart. It can be found in [update_groups.rs](../../../stats/src/update_groups.rs) or
+in [split.py (variable `update_groups_mapping`)](./split.py). The group name must end with __GROUP, for example:
```
Comment (bragov4ik) [inline stats/config/migration/m20240618_080000_update_groups_and_separate_layout/README.md:77]:
```suggestion
in [split.py (variable `update_groups_mapping`)](./split.py). Usually, the group names end with `...__GROUP`, for example:
```

File: stats/config/migration/m20240618_080000_update_groups_and_separate_layout/README.md [hunk 2]
```diff
@@ -73,8 +73,17 @@ Counters:
...
+```
+STATS_UPDATE_GROUPS__SCHEDULES__TOTAL_BLOCKS_GROUP
+STATS_UPDATE_GROUPS__SCHEDULES__ACTIVE_ACCOUNTS_GROUP
+STATS_UPDATE_GROUPS__SCHEDULES__COMPLETED_TXNS_GROUP
+STATS_UPDATE_GROUPS__SCHEDULES__TOTAL_ADDRESSES_GROUP
+STATS_UPDATE_GROUPS__SCHEDULES__NEW_TXNS_GROUP
+STATS_UPDATE_GROUPS__SCHEDULES__NEW_CONTRACTS_GROUP
+STATS_UPDATE_GROUPS__SCHEDULES__NEW_VERIFIED_CONTRACTS_GROUP
```
Comment (bragov4ik) [inline stats/config/migration/m20240618_080000_update_groups_and_separate_layout/README.md:85]:
```suggestion
STATS_UPDATE_GROUPS__SCHEDULES__ACTIVE_ACCOUNTS_GROUP
STATS_UPDATE_GROUPS__SCHEDULES__NEW_ACCOUNTS_GROUP
```
no need to list that much envs :) let's keep it concise

**Review by bragov4ik — CHANGES_REQUESTED** [review]

**Comment by bragov4ik** [comment]
Thanks for suggestions! It's definitely better now

**Comment by InoMurko** [comment]
> Thanks for suggestions! It's definitely better now Please consider my comments and I'll merge the patch

cool! merged your suggestions.

**Review by bragov4ik — APPROVED** [review]

**Pull request merged**